Fix up round robin description (#1295)
sensu/sensu-docs: content/sensu-go/5.3/reference/checks.md

|round_robin |      |
-------------|------

> description  | Round-robin check scheduling lets you distribute proxy check executions evenly over a group of Sensu agents. When set to `true`, Sensu executes the proxy check using each agent within the subscription in turn.

Reviewer: ```suggestion
description  | Round-robin check scheduling lets you distribute check executions evenly over a group of Sensu agents. When set to `true`, Sensu executes the check on each subscribing agent in turn.
```
